PowerPointLabs/PowerPointLabs/ActionFramework/PictureSlidesLab/PictureSlidesEnabledHandler.cs

```diff
@@ -0,0 +1,16 @@
+﻿using PowerPointLabs.ActionFramework.Common.Attribute;
+using PowerPointLabs.ActionFramework.Common.Interface;
+using PowerPointLabs.TextCollection;
+using PowerPointLabs.TimerLab;
+
+namespace PowerPointLabs.ActionFramework.TimerLab
```

Comment:
I'm guessing you took this from TimerLab... Probably should change the namespace? :P

Reply:
Changed!